docs/get-started/alerting.md

```diff
@@ -129,48 +227,65 @@ templates:
 2. On the **Create alert rule** page, select the **Percona templated alert** option. If you want to learn about creating Grafana alerts instead, check our [Grafana's documentation](https://grafana.com/docs/grafana/latest/alerting/).
 3. In the **Template details** section, choose the template on which you want to base the new alert rule. This automatically populates the **Name**, **Duration**, and **Severity** fields with information from the template. You can change these values if you want to override the default specifications in the template.
-4. In the **Filters** field, specify if you want the alert rule to apply only to specific services or nodes. For example: `service_name'`, Operator:`MATCH`, VALUE: `ps5.7`.
+4. In the **Filters** field, specify if you want the alert rule to apply only to specific services or nodes. For example `service_name'`, Operator:`MATCH`, VALUE: `ps5.7`.
 5. From the **Folder** drop-down menu, select the location where you want to store the rule.
 6. Click **Save and Exit** to close the page and go to the **Alert Rules** tab where you can review, edit and silence your new alert.
 
 ## Silence alerts
-Create a silence when you want to stop notifications from one or more alerting rules.
+Create a silence when you want to stop notifications from one or more alerting rules for a very specific amount of time. Silences default to today’s current date and  have a default duration of two hours.
 
-Silences stop notifications from being sent to your specified contact points.
+During a silence, PMM continues to track metrics but does not trigger alerts or send notifications to any specified contact points. Once the silence expires alerts and notifications will resume.
+
+For example, you can create a silence to suppress a notifications for a specific period of time.
 
 Silenced alerts are still recorded under **Alerting > Fired Alerts** so that you can review them later. Silenced alerts are disabled for as long as it's specified in the Silence Duration or until you remove a silence.
 
-For information on creating silences, see [About alerting silences](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/create-silence/) in the Grafana documentation.
+#### Add a silences
+
+You can easily silence an alert from the **Fired alerts** page or from the **Alert rules** page.
+You can also create a silence from the **Silences** page, but here you would also need to define labels that match the alert that you want to silence.
+
+To create a new silence from the **Silences** page:
+
+1. Click the **New Silence** button.
+2. Select the start and end date to indicate when the silence should go into effect and expire.
+3. Optionally, update the duration to alter the time for the end of silence in the previous step to correspond to the start plus the duration.
+4. Enter one or more matching labels by filling out the **Name** and **Value** fields. Matchers determine which rules the silence will apply to.
```

Comment:
I would add: "You must enter at least 1 valid set of matching labels, or the system will not allow you to complete creating the silence. This is because if the system allowed you to create a silence without any valid matchers, it would expect that you want to silence ALL ALERTS."